src/pages/tasks/edit/Edit.tsx

```diff
@@ -89,7 +96,9 @@ export function Edit() {
       navigationTopRight={task && <Actions task={task} />}
       onSaveClick={() => task && handleSave(task)}
     >
-      {task && <TaskDetails task={task} handleChange={handleChange} />}
+      {task && (
+        <TaskDetails task={task} handleChange={handleChange} errors={errors} />
```

Comment:
With this change, I actually included errors as response to my part of the function and the part of the function you implemented earlier. If I include errors as an prop to `TaskDetails`, I have to implement it in every component where we use this component. So that's the reason for the code change here.